src/app/components/memberInfoForm/memberInfoFormComponent.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { browserHistory } from 'react-router';
+import { Field, reduxForm } from 'redux-form';
+
+const MemberInfoForm = ({ editting, deleteMember, handleSubmit }) => (
+  <form onSubmit={ editting, deleteMember, handleSubmit }>
+    <h3>Info</h3>
+    <Field name="firstName" component="input" type="text" />
+    <Field name="lastName" component="input" type="text" />
+    <Field name="email" component="input" type="text" />
+    <Field name="phone" component="input" type="text" />
+    <h3>Role</h3>
+    <p className="list-item"><Field name="role" component="input" type="radio" value="regular" />Regular - Can't delete members</p>
```

Comment:
It would be nice if clicking on the text would toggle the radio button. This can be achieved with the `<label>` tag in HTML, although I'm not sure how that interacts with the `Field` component you're using here.